proto/pocket/supplier/tx.proto

```diff
@@ -20,4 +22,13 @@ message MsgStakeSupplier {
 
 message MsgUnstakeSupplierResponse {}
 
-message MsgStakeSupplierResponse {}
\ No newline at end of file
+message MsgStakeSupplierResponse {}
+
+message MsgSubmitProof {
+  string                       supplier_address = 1;
+  pocket.session.SessionHeader session_header   = 2;
+  bytes                        proof            = 3;
```

Comment:
Not a blocker. Did we decide not use an explicit type from the SMT repo?

Reply:
Correct.

> We can continue to embed this entire struct in the protobufs using proof.Marshal() and store it as a []byte

--[@h5law](https://github.com/pokt-network/poktroll/pull/44#discussion_r1352406912)

Reply:
My only request is to leave a comment explaining that the proof is a serialized version of `smt.XXX`